examples/library/test_manycore_eva/main.c

```diff
@@ -166,7 +165,7 @@ int test_manycore_eva () {
                         tgt_epa = rand() % DRAM_EPA_SIZE; // Small, but we'll deal.
 
                         tgt_x = (rand() % (dim_x - 1)) + origin_x;
-                        tgt_y = hb_mc_config_get_dram_y(config);
+                        // tgt_y = hb_mc_config_get_dram_y(config);
```

Comment:
No y coordinates required for DRAM pods?

Reply:
There is, but it's handled in a different way. This is @mrutt92's least favorite test

Reply:
This test suuuuuuuuuuuxxxxxxxxxx. When it fails, 99 times out of 100 the solution is to fix the test instead of any RTL or library code.

Reply:
We use these now with pods https://github.com/bespoke-silicon-group/bsg_replicant/blob/c06120ac2dd477aedb88973a709bb1d81ccc8101/libraries/bsg_manycore_config_pod.h#L269.

Reply:
Ahh okay.